tests/testthat/test-http.R

```diff
@@ -62,7 +60,7 @@ setup({
   Sys.sleep(2)
 })
 
-teardown({
+withr::defer({
```

Comment:
When is this `withr::defer` triggered? Does `testthat` have an environment for each test file, meaning the `defer` runs as the file completes?

Reply:
At the end of the file, and yes 😄